lib/ecto/json.ex

```diff
@@ -10,12 +10,14 @@ if Code.ensure_loaded?(Jason.Encoder) do
           Repo.preload(#{inspect(owner)}, #{inspect(field)})
 
       Or choose to not encode the association when converting the struct \
-      to JSON by explicitly listing the JSON fields in your schema:
+      to JSON by explicitly listing the JSON fields in your schema, or by \
+      excluding it (commented out):
 
           defmodule #{inspect(owner)} do
             # ...
 
             @derive {Jason.Encoder, only: [:name, :title, ...]}
+            # @derive {Jason.Encoder, except: [#{inspect(field)}]}
             schema ... do
```

Comment:
One alternative here is to show two separate examples without having a directive to uncomment a line.

```elixir
    @doc """


      Or choose to not encode the association when converting the struct \
      to JSON by explicitly listing the JSON fields in your schema:

          defmodule #{inspect(owner)} do
            # ...
            @derive {Jason.Encoder, only: [:name, :title, ...]}
            schema ... do

      Or choose to not encode the association by excluding it in your schema:

          defmodule #{inspect(owner)} do
            # ...
            @derive {Jason.Encoder, except: [#{inspect(field)}]}
            schema ... do

 
    """
```

Reply:
I'm happy to do convert to that style, if that's preferred (also happy if maintainers want to adjust this in that direction). The only thing to add is that this isn't part of narrative documentation, this is traceback you will get if your code causes this error, so that's why I was trying to lean towards being more terse.